lib/kadalulib.py

```diff
@@ -47,10 +47,11 @@ def is_gluster_mount_proc_running(volname, target_path):
     Check if glusterfs process is running for the given Volume name
     to confirm Glusterfs process is mounted
     """
+    # TODO: see if you can use '-w' instead
     cmd = (
         r'ps ax | grep -w "/usr/sbin/glusterfs" '
-        r'| grep "\-\-volfile\-id %s" '
-        r'| grep -q "%s"' % (volname, target_path)
+        r'| grep "\-\-volfile\-id %s " '
+        r'| grep -q " %s"' % (volname, target_path)
```

Comment:
how space before helps? We can use `-w` or `egrep -q "%s$" % target_path`

Reply:
Also can be changed to `mountpath`

Reply:
Done!